purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/PaywallColor.kt

```diff
@@ -24,6 +25,12 @@ data class PaywallColor(
     @RequiresApi(Build.VERSION_CODES.O)
     val underlyingColor: Color?,
 ) {
+    /**
+     * The color converted to a ColorInt
+     */
+    @ColorInt
+    val colorInt: Int = Color.parseColor(stringRepresentation)
```

Comment:
I was thinking `intRepresentation` might make more sense, but if `@ColorInt` is an Android thing then maybe this name makes sense.

Reply:
Well... I think either could work... But yeah, I kinda feel `colorInt` makes more sense, since it represents that... Lmk if you feel strongly though.